cg_openmm/utilities/iotools.py

```diff
@@ -130,42 +130,31 @@ def write_pdbfile_without_topology(CGModel, filename, energy=None):
                     str(
                         "ATOM"
                         + str("{:>7}".format(bead_index))
-                        + " X"
+                        + "  X"
                         + str(element_index)
-                        + str("{:>6}".format(str("MT")))
+                        + str("{:>5}".format(str("MT")))
                         + " A"
                         + str("{:>4}".format(monomer_index + 1))
-                        + "     "
+                        + "    "
                         + str(
-                            "{:>7}".format(
-                                round(
-                                    coordinates[bead_index - 1][0]
-                                    .in_units_of(unit.angstrom)
-                                    ._value,
```

Comment:
For cleanness, should consider using value_in_unit(unit.angstrom) way of writing a Quantity in specific units, rather than using lower level functions.

Reply:
Consider switching to f-string formatting.  I think this whole thing could be a bit simpler.  

print(f"ATOM{bead_index:>7d} X X {element_index:d}      MT     MT     A  {monomer_index:>4d}    {coordinates[bead_index-1][0].value_in_units(unit.angstrom):8.3f})

Could probably use list comprehension to do the three coordinates, see https://www.reddit.com/r/learnpython/comments/918i8c/trying_to_format_a_list_of_floats_with_fstrings/

Reply:
Ok I condensed a lot of this using f string formatting. In my opinion the list comprehension isn't needed here, but it is much better than before. Changed to use value_in_units too.